chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -18,6 +22,29 @@
 #undef RenderViewContextMenu
 #define RenderViewContextMenu RenderViewContextMenu_Chromium
 
+namespace {
+
+GURL GetSelectionNavigationURL(Profile* profile, const base::string16& text) {
+  AutocompleteMatch match;
+  AutocompleteClassifier classifier(
+      std::make_unique<BraveAutocompleteController>(
+          std::make_unique<BraveAutocompleteProviderClient>(profile),
+          nullptr, AutocompleteClassifier::DefaultOmniboxProviders()),
+      std::make_unique<BraveAutocompleteSchemeClassifier>(profile));
+  classifier.Classify(text, false, false,
+                      metrics::OmniboxEventProto::INVALID_SPEC, &match, NULL);
+  classifier.Shutdown();
+  return match.destination_url;
+}
+
+}  // namespace
+
+#define BRAVE_APPEND_SEARCH_PROVIDER \
+  selection_navigation_url_ = \
```

Comment:
Do we need to overwrite the result even for normal profile? I thought we only need it for OTR profile.

Reply:
Yes, it's redundant. fixed.